test/IntegrationTests/StrongNamedTests.cs

```diff
@@ -34,13 +34,15 @@ public StrongNamedTests(ITestOutputHelper output)
     public async Task SubmitsTraces()
     {
         using var collector = await MockSpansCollector.Start(Output);
+        SetExporter(collector);
         collector.Expect("TestApplication.StrongNamedValidation");
 
         var assemblyPath = GetTestAssemblyPath();
         var integrationsFile = Path.Combine(assemblyPath, "StrongNamedTestsIntegrations.json");
         File.Exists(integrationsFile).Should().BeTrue();
         SetEnvironmentVariable("OTEL_DOTNET_AUTO_INTEGRATIONS_FILE", integrationsFile);
-        RunTestApplication(otlpTraceCollectorPort: collector.Port);
+        SetEnvironmentVariable("CORECLR_ENABLE_PROFILING", "1"); // uses bytecode instrumentation
```

Comment:
nit: As it's mostly constant, maybe do it similarly like `SetExporter()`. For example `EnableByteCodeInstrumentations()`?

Reply:
I was considering it, but I was not sure if it would not be "over abstraction" 😉 I can address it in a separate PR.

Reply:
I think that this usage is common enough that it's worth having this abstraction so that we can minimize the chance of someone typing the environment variable name incorrectly. I agree that making such a change should be done in a separate PR.